src/BUILD

```diff
@@ -0,0 +1,11 @@
+filegroup(
+    name = "core",
```

Comment:
"core_srcs" or "runtime_srcs" would be a more typical name

Reply:
Isn't that a bit chicken chicken? - `deps = ["//src:core_src"]`

Reply:
That's just because we have a folder named src... The general convention is for BUILD rules with "normal" names to be proper buildable targets. Filegroups are usually named "resources" or "srcs", etc. `//src::core` looks to me like it's gonna build the whole core library, which it's not..

Reply:
Fair enough; updated.